README.md

```diff
@@ -23,6 +23,7 @@ The plugin will log activity and any errors to the Eclipse *Error Log* view (`Wi
 
 The aforementioned usage will import the projects on eclipse startup and will utimately place you in the IDE.
 To perform a headless-import that does not actually start the IDE, run with `-application com.seeq.eclipse.importprojects.headlessimport`.
+On Windows use `eclipsec` insted of `eclipse` launcher.
```

Comment:
Minor spelling mistake: `insted` should be `instead`.

Reply:
indeed. fixed now